docs/layers/identity/centralized-terraform-access.mdx

```diff
@@ -60,8 +218,55 @@ Follow the Identity Providers documentation for adding a SAML login.
 
 With AWS SAML, we create a federated SAML login that connects to the "team" in the identity account, and then users can assume other roles from there. We use the [AWS Extend Switch Roles plugin](https://github.com/tilfinltd/aws-extend-switch-roles) that makes this much easier, but it's not as intuitive as Identity Center.
 
-:::tip AWS IAM Identity Center or AWS SAML? Which do I choose?
+```mermaid
+flowchart TB
+    subgraph "Human Access Architecture with AWS SAML"
+        subgraph "core-identity"
+            subgraph aws_teams["aws-teams"]
+                devops_teams["devops"]
+                developers_teams["developers"]
+                billing_teams["billing"]
+            end
+        end
 
-The vast majority of our customers prefer AWS IAM Identity Center (SSO). The convenience of a web console login is hard to beat. However, some customers prefer SAML for its simplicity and compatibility with existing systems. We support both methods, and you can choose the one that best fits your needs.
+        subgraph "plat-prod"
+            subgraph aws_team_roles_prod["aws-team-roles"]
+                admin_prod["admin"]
+                billing_prod["billing"]
+                reader_prod["reader"]
+            end
+        end
 
-:::
+        subgraph "plat-sandbox"
+            subgraph aws_team_roles_sandbox["aws-team-roles"]
+                admin_sandbox["admin"]
+                billing_sandbox["billing"]
+                reader_sandbox["reader"]
+            end
+        end
+
+        g_suite["GSuite\nFederated Identity"] -->|SAML| devops_teams
+        g_suite -->|SAML| developers_teams
+        g_suite -->|SAML| billing_teams
+
+        developers_teams --> reader_prod
+        devops_teams --> admin_prod
+        billing_teams --> billing_prod
+
+        developers_teams --> admin_sandbox
+        devops_teams --> admin_sandbox
+        billing_teams --> billing_sandbox
+    end
+```
+
+### How do I use AWS IAM Identity Center (SSO)?
+
```

Comment:
Start with a short answer.

AWS IAM Identity Center (SSO) is natively integrated with the AWS Web Console and the AWS CLI. You don't need to do anything special to get started with it, once it's been properly configured.

...Then how it works.